### PR TITLE
All all-srcs to test-infra. Add verify-boilerplate test target.

### PR DESCRIPTION
**Pull request by spxtr**
We can also do gofmt, go vet, and pylint, but lets start with just the one.
```
py_test(
    name = "verify-boilerplate",
    srcs = ["verify-boilerplate.py"],
    data = ["//:all-srcs"],
)
```

### CODE REVIEW

File: verify/BUILD [hunk 1]
```diff
@@ -0,0 +1,19 @@
+py_test(
```
Comment (ixdy) [inline verify/BUILD]: maybe add
```
package(default_visibility = ["//visibility:public"])

licenses(["notice"])
```
to this file?

Comment (spxtr) [inline verify/BUILD]: What does that actually do?

Comment (spxtr) [inline verify/BUILD]: (licenses, I mean)

Comment (spxtr) [inline verify/BUILD]: https://bazel.build/versions/master/docs/be/functions.html#licenses

Comment (ixdy) [inline verify/BUILD]: https://bazel.build/versions/master/docs/be/functions.html#licenses

It probably doesn't really matter in this repo. I think it mostly matters when you link code with more restrictive licensing rules.

Comment (spxtr) [inline verify/BUILD]: OK, done.

File: .gazelcfg.json [hunk 2]
```diff
@@ -2,5 +2,6 @@
 	"GoPrefix": "k8s.io/test-infra",
 	"SkippedPaths": [
 		"jenkins/aws-janitor"
-	]
+	],
+    "AddSourcesRules": true
```
Comment (ixdy) [inline .gazelcfg.json]: nit: indent is weird here

Comment (spxtr) [inline .gazelcfg.json]: tabs...

**Pull request merged**